util/src/main/java/io/camunda/zeebe/util/health/HealthMonitorable.java

```diff
@@ -10,13 +10,24 @@
 /** Any component that can be monitored for health should implement this interface. */
 public interface HealthMonitorable {
 
+  /**
+   * Used by a HealthMonitor to get the name of this component. Most `HealthMonitorable`s override
+   * this method to return their actor name. If they don't override the method, the name of the
+   * class is returned.
+   *
+   * @return the name of this component
+   */
+  default String getName() {
```

Comment:
:+1: 

:x: When a subcomponenent is unhealthy, this is how it is shown in the log.

`
 Detected 'UNHEALTHY' components. The current health status of components: [CriticalComponentsHealthMonitor{status=UNHEALTHY, issue=Broker-0-LogStream-1{status=UNHEALTHY, issue=Broker-0-LogAppender-1{status=UNHEALTHY, issue=java.lang.RuntimeException: FIRE!!!!!}}}]
`

Here `CriticalComponentHealthMonitor` is a monitor with in a specific partition. So can we give a proper name for it? It should be something like `partition-<ID>`. So the log looks like:
`
 Detected 'UNHEALTHY' components. The current health status of components: [Partition-1{status=UNHEALTHY, issue=Broker-0-LogStream-1....]
`